sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsClient.cs

```diff
@@ -19,7 +20,8 @@ namespace Azure.AI.TextAnalytics
     public class TextAnalyticsClient
     {
         private readonly Uri _baseUri;
-        internal readonly TextAnalyticsRestClient _serviceRestClient;
+        internal readonly AnalyzeTextRestClient _serviceRestClient;
+        internal readonly MicrosoftCognitiveLanguageServiceRestClient _cognitiveRestClient;
```

Comment:
what is the difference between these two? I don't see the `MicrosoftCognitiveLanguageServiceRestClient` referenced here